tests/python/PEP8NotebookBearTest.py

```diff
@@ -0,0 +1,133 @@
+from tests.LocalBearTestHelper import verify_local_bear
+from bears.python.PEP8NotebookBear import PEP8NotebookBear
+
+
+def prepare_string(file):
+    """
+    Utility function to prepare Notebook strings for further processing.
+
+    replace() necessary to avoid
+    nbformat.reader.NotJSONError: Notebook does not appear to be JSON
+    which is caused by
+    json.decoder.JSONDecodeError: Invalid control character
+    which is caused by the \n characters inside the input cells
```

Comment:
This is not only interesting, it is also annoying. ;)
It does not happen when you read the file directly (i.e. when you pass a file directly to the bear).
If you know a better solution I am happy to change this.